lib/order.rb

```diff
@@ -0,0 +1,36 @@
+require 'dotenv/load'
+require_relative 'menu'
+require_relative 'sms'
+
+class Order
+  attr_reader :current_total
+
+  def initialize
```

Comment:
Did you try injecting the Menu and Sms classes here?

Reply:
Thanks! I hadn't thought about that before.